app/javascript/components/menus/new/base-available-recipes.vue

```diff
@@ -3,6 +3,10 @@
     <div class="min-w-min h-10 bg-gray-500 border border-gray-500 self-stretch flex-grow-0 py-2 px-6 font-hind font-normal text-white text-base">
       Recetas
     </div>
+    <recipe-card />
```

Comment:
Creo que es mejor dejar estas con un for, pero como por ahora no tienen información real no importa tanto

Reply:
Después cuando lo conectemos con el _backend_ de todas maneras irá con un for